src/routes/media.rs

```diff
@@ -67,11 +67,41 @@ fn process_file_upload(
     }
 }
 
+fn get_foreign_key(
+    key_name: &'static str,
```

Comment:
why is this a static reference?

Reply:
Because it's a string literal. I could make it a regular slice but why bother?